documentation/Deployment-Guide.md

```diff
@@ -0,0 +1,1137 @@
+# Microsoft
+
+# Securiy Surveillance
```

Comment:
Spell Correction Required: Security

Reply:
Updated the document with correct spelling.